ios/brave-ios/Sources/BraveWallet/Crypto/Stores/AccountsStore.swift

```diff
@@ -33,7 +33,7 @@ class AccountsStore: ObservableObject, WalletObserverStore {
 
   let currencyFormatter: NumberFormatter = .usdCurrencyFormatter
 
-  /// Cache of token balances for each account. [account.address: [token.id: balance]]
+  /// Cache of token balances for each account. [account.cacheBalanceKey: [token.id: balance]]
   private var tokenBalancesCache: [String: [String: Double]] = [:]
```

Comment:
Is there a thing like branded types in typescrpit but for swift?

Reply:
i think thats `typealias` in swift. i will update. thanks!